src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -80,26 +80,33 @@ To deploy these resources to your cloud sandbox, run the following CLI command i
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
-```bash
+```bash title="Terminal" showLineNumbers={false}
 npx amplify sandbox
 ```
 
 </InlineFilter>
 
 <InlineFilter filters={["android"]}>
 
-```bash
-npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_src/main/res/raw_folder>
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_android_project>/src/main/res/raw/
 ```
 
 </InlineFilter>
 
 <InlineFilter filters={["swift"]}>
 
-```bash
+```bash title="Terminal" showLineNumbers={false}
 npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_swift_project>
 ```
 
+</InlineFilter>
+<InlineFilter filters={["flutter"]}>
+
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox --config-format dart --config-out-dir <path_to_flutter_project>/lib/
```

Comment:
Same as above

Aren't we running this in the rooth of the project? Then we can run the following.
```suggestion
npx amplify sandbox --config-format dart --config-out-dir lib
```